modules/benchmark/R/load.netcdf.R

```diff
@@ -50,19 +50,19 @@ load.x_netcdf <- function(data.path, format, site, vars=NULL){
     # throw error if can't parse time format  
     if(is.na(date.origin)) logger.error("All time formats failed to parse. No formats found.")
     
-    time.stamp.match <- gsub("UTC", "", date.origin)
+    time.stamp.match <- gsub("UTC", "", date.origin) 
     t.units <- gsub(paste0(" since ", time.stamp.match, ".*"), "", ncatt_get(nc[[i]], dims[time.var])$units)
     
     
```

Comment:
according to stackoverflow there is a lot of story behind it :) https://tools.ietf.org/html/rfc3092
here writing these in two lines just helped me while testing and debugging, could be reduced to one line indeed